Yoast/wordpress-seo
# Improve banners in admin

## Summary

New look for Premium page:

<img width="1115" alt="Screen Shot 2019-10-30 at 22 43 30" src="https://user-images.githubusercontent.com/487629/67901293-c745ff00-fb66-11e9-86e5-d8e09783bc90.png">

New look for sidebar:

<img width="1856" alt="Screen Shot 2019-10-30 at 22 50 22" src="https://user-images.githubusercontent.com/487629/67901904-3b34d700-fb68-11e9-8047-fc05baa2c254.png">


This PR can be summarized in the following changelog entry:

* Updated banners to new assistant style.

## Relevant technical choices:

* Made some styling changes to the Premium page, to make them use cards instead of rounded borders
* Switched the banner images on the Premium page out for assistants.
* Used those same assistants in the ad sidebar on admin pages.
* Removed all the no longer needed images.
* Removed the option to remove the bottom banner on admin pages. 
* Switched to native fonts.
* Made all the banners have a "card" look.
* Changed the three Academy banners to one simple Academy card.
* Changed the wording on the plugin banners.

Would like to get feedback on this from @jono-alderson @michielheijmans, as it does change the look on some rather important banners, but I feel this doesn't diminish them.

## Test instructions

This PR can be tested by following these steps:

* Checkout, build CSS, see the changes.

## UI changes

## Documentation

## Quality assurance


Fixes #
